lib/gstreamer/va/util.py

```diff
@@ -44,6 +44,15 @@ def map_best_hw_format(format, hwformats):
     match_best_format(
       format, set(hwformats) & set(get_supported_format_map().keys())))
 
+@memoize
+def map_deinterlace_method(method):
+  return {
+    "bob"               : "bob",
+    "motion-adaptive"   : "adaptive",
+    "motion-compensated": "compensated",
+    "none"              : "none"
```

Comment:
vadeinterlace does not support `none` value. remove.

Reply:
updated